_docs_use/establish-a-new-digital-relationship-to-an-organization.md

```diff
@@ -0,0 +1,23 @@
+---
+# !!! Warning: Do not edit this file; any changes must be replicated in Excel !!!
+permalink: use/establish-a-new-digital-relationship-to-an-organization
+published: false
+title: "establish a new digital relationship to an organization"
+type: scenario
+toc: true
+properties:
+  - id: SC017
+  - category: Get in touch with other Identities
+  - description: Nutzer möchte ohne einen Online Account einen neuen Vertrag mit einer Organisation eingehen
```

Comment:
Deutsch / Englisch?

Reply:
Das ist Resultat des Skripts, was aus den Szenarien und Use Cases aus der Doku Excel Tabelle diese Dateien generiert. Man müsste also checken, dass man die Sprache in der Tabelle einheitlich hält.  Ist aber im Prinzip separat von meinen Änderungen an diesem PR und würde ich deshalb im Nachhinein anpassen, falls gewollt.

Reply:
Passt, kann weg 👍

Reply:
Ich resolve auch mal die anderen Kommentare, die mit der Excel Tabelle zusammenhängen.